lib/analytics.js

```diff
@@ -59,6 +59,17 @@ function Analytics() {
 }
 
 /**
+ * Used for stubbing navigator.doNotTrack in the unit tests.
+ */
+
+Analytics._doNotTrack = function() {
+  // Fallback to window.doNotTrack in older browsers
+  var doNotTrack = navigator.doNotTrack || window.doNotTrack;
+  // Normalize values (Firefox can return 'unspecified')
+  return doNotTrack === '1' ? '1' : '0';
```

Comment:
Why don't we just return a boolean here?
`return doNotTrack === '1';`

Simplifies line 120 as well:
```
var doNotTrack = options.doNotTrack && Analytics._doNotTrack();
```